Core/clim-core/panes.lisp

```diff
@@ -913,10 +913,11 @@ which changed during the current execution of CHANGING-SPACE-REQUIREMENTS.
       (error 'sheet-supports-only-one-child :sheet pane))
     (sheet-adopt-child pane (first contents))))
 
-(defmethod compose-space ((pane single-child-composite-pane) &key width height)
+(defmethod compose-space ((pane single-child-composite-pane)
```

Comment:
Commit message: "If they were not supplied then width and height were *defaulted* NIL *which* lead to an error in make-space-requirements."

Otherwise: :+1: :+1: :smiley: :smiley:

Reply:
fixed